tools/odbclient/src/odbclient/odbclient.py

```diff
@@ -49,12 +49,16 @@ class OdbClient:
         The path to the odb file
 
     abaqus_bin : string, optional
-        The path to the abaqus *binary* (not a .bat or shell script).
-        Guessed if not given.
+        The path to the abaqus *binary* (.exe, not .bat or a shell script).
```

Comment:
I think it actually does work with a shell script.  At least with bash. But we probably shouldn't encourage it. 

`.exe` is Windows only, though. So maybe just `(no .bat or shell script)`